From: =?utf-8?q?Pablo_Correa_G=C3=B3mez?= <ablocorrea@hotmail.com>
Date: Mon, 14 Feb 2022 21:08:01 +0100
Subject: [PATCH] removable-media: Use a child for GtkCheckButton label
 to wrap it

It allows the panel to fit in narrow displays. The child property
for GtkCheckButton will only be available since 4.8

[1] https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4489

Origin: https://gitlab.gnome.org/pabloyoyoista/gnome-control-center/-/commit/08bf9f77c2d97498d8361e91c833d4b5478ef62e
---
 panels/removable-media/cc-removable-media-panel.ui | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/panels/removable-media/cc-removable-media-panel.ui b/panels/removable-media/cc-removable-media-panel.ui
index 5eba33b..8965e02 100644
--- a/panels/removable-media/cc-removable-media-panel.ui
+++ b/panels/removable-media/cc-removable-media-panel.ui
@@ -191,8 +191,13 @@
               <object class="GtkCheckButton" id="autorun_never_checkbutton">
                 <property name="valign">end</property>
                 <property name="vexpand">False</property>
-                <property name="use_underline">True</property>
-                <property name="label" translatable="yes">_Never prompt or start apps on media insertion</property>
+                <property name="child">
+                  <object class="GtkLabel">
+                    <property name="use_underline">True</property>
+                    <property name="wrap">True</property>
+                    <property name="label" translatable="yes">_Never prompt or start apps on media insertion</property>
+                  </object>
+                </property>
               </object>
             </child>
           </object>
